### PR TITLE
add new (non production) keyboard in hand-wired section 

### PR DESCRIPTION
**Pull request by greenjack-z**
## Description

add the keyboard to the list of supported ones - to be able follow the work with the QMK Configurator

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
Please follow the PR checklist - there are a number of outdated conventions in this PR.

**Comment by greenjack-z** [comment]
> Please follow the PR checklist - there are a number of outdated conventions in this PR.
done, as far as I understood.

File: keyboards/handwired/zergo/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# zErgo
```
Comment (fauxpark) [inline keyboards/handwired/zergo/readme.md:1]: This file needs to follow the template.

File: keyboards/handwired/zergo/makefile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ifndef MAKEFILE_INCLUDED
```
Comment (fauxpark) [inline keyboards/handwired/zergo/makefile:1]: Please delete this file.

File: keyboards/handwired/zergo/keymap.c [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+		
+	}
+
+	if (usb_led & (1 << USB_LED_KANA)) {
+		
+	} else {
+		
+	}
+
+}
```
Comment (fauxpark) [inline keyboards/handwired/zergo/keymap.c:202]:
```suggestion
```

File: keyboards/handwired/zergo/zergo.h [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define KEYMAP( \
```
Comment (fauxpark) [inline keyboards/handwired/zergo/zergo.h:21]:
```suggestion
#define LAYOUT( \
```

File: keyboards/handwired/zergo/keymap.c [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	KEYMAP(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS)
+
```
Comment (fauxpark) [inline keyboards/handwired/zergo/keymap.c:148]:
```suggestion
```

File: keyboards/handwired/zergo/keymap.c [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "zergo.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```
Comment (fauxpark) [inline keyboards/handwired/zergo/keymap.c:21]:
```suggestion
	LAYOUT(
```

File: keyboards/handwired/zergo/keymap.c [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+		KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, 
+		KC_HOME, KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_PGUP, 
+		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_5, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_BSLS, 
+		KC_LCTL, KC_A, KC_S, KC_D, KC_F, KC_G, KC_RBRC, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_LCTL, 
+		MT(MOD_LSFT, KC_END), KC_UP, KC_Z, KC_X, KC_C, KC_V, KC_ENT, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, MT(MOD_LSFT, KC_PGDN), 
+		KC_LEFT, KC_DOWN, KC_RGHT, KC_LALT, KC_SPC, KC_BSPC, KC_SPC, KC_RALT, KC_LGUI, KC_RGUI, LT(1, KC_NO)),
+
+	KEYMAP(
```
Comment (fauxpark) [inline keyboards/handwired/zergo/keymap.c:29]:
```suggestion
	LAYOUT(
```

File: keyboards/handwired/zergo/keymap.c [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "zergo.h"
```
Comment (fauxpark) [inline keyboards/handwired/zergo/keymap.c:17]:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/handwired/zergo/rules.mk [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+
+# Bootloader selection
+BOOTLOADER = halfkay
+
+# Build Options
+#   change yes to no to disable
+#
+
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/handwired/zergo/rules.mk:12]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/handwired/zergo/readme.md (hunk 1 above)
Comment (greenjack-z) [reply keyboards/handwired/zergo/readme.md:1]: I've check and add the image

File: keyboards/handwired/zergo/readme.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
...
+
+![zergo](https://lh3.googleusercontent.com/ebufdZluIsuej_2MMqICxgyStO1TVOQXD7g-Nb4iJhjHOiG_g3gAAH335jnDCo7crZJPiUMTLWUrQ8qJWk9wbCAbMjEaNanWuqNdkMfQ7h0Oq2AYgcDvVulS68AOG65g2vtjp1_wBaJjy2D0SqGoEibhxi5A9ehQxHUJmCUaTuNiHXGR02shFr8JF6wgP7K72OY4g58RKyOUnCDRoiYENuBcZzcT_dR2Zbmzyj9fqo7VBuMJv3ry4S1aM1JiUw2ctMabhAYbjFcp7DqhgGbl0bTbcQSpYiywDLyeJol7zU0tCgqT8KixXn1YbwphlgYBhw379KiMoBHuFzoQNjwsWRPK0lzgsNRE9j6KrCCwVEokSdrt9eTLHO5emM--J_F32JuF-eT__pQBBki0kU0nS0TZl-4TWY5L8L7cb8Awg-OEX4mxN_ONWAkxu3eSKuKPOWYUVwf-VVDhjHjfv74x8Eq84T4jqyXsMPmBZjmi_Xz5c2KUusC0fQYAJa5bP3FLjEd83IBA4n-B4fx1nVUtKZAZCkVXWgJNRrYRjZlY5kKHPfu9w-2T6Q_vLwNfr5FudEHlxfTAjecxGoFtgnRmCas1ZSf0UZ15FIAfO22mQlmiRdGZge_OXn4UuWOCLlRviBiZJTIbToKaYpgqlkqlg0j7nqx951xSCEMBjEL_GDi1IFZdm6RvQAPp=w830-h358-no?authuser=0)
+
+A compact 70% (15x6) keyboard inspired by the Katana60 project, but with F keys and adopted for the Russian layout.
+
+* Keyboard Maintainer: [Greenjack](https://github.com/greenjack-z)
```
Comment (fauxpark) [inline keyboards/handwired/zergo/readme.md:7]:
```suggestion
* Keyboard Maintainer: [Greenjack](https://github.com/greenjack-z)
* Hardware Supported: *The PCBs, controllers supported*
* Hardware Availability: *Links to where you can find this hardware*

Make example for this keyboard (after setting up your build environment):

    make handwired/zergo:default

Flashing example for this keyboard:

    make handwired/zergo:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Please fill out the Hardware supported and Hardware availability items.

Comment (greenjack-z) [reply keyboards/handwired/zergo/readme.md:7]: It's a prototype keyboard made by me using teensy 2.0++ controller and self made PCB - what exactly I have to list?

File: keyboards/handwired/zergo/info.json [hunk 11]
```diff
@@ -0,0 +1,93 @@
+{
+    "keyboard_name": "zergo", 
+    "url": "", 
+    "maintainer": "greenjack", 
+    "width": 15, 
+    "height": 6,
```
Comment (noroadsleft) [inline keyboards/handwired/zergo/info.json:6]:
```suggestion
    "height": 6.25,
```

File: keyboards/handwired/zergo/readme.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
+# zErgo
+
+![zergo](https://lh3.googleusercontent.com/ebufdZluIsuej_2MMqICxgyStO1TVOQXD7g-Nb4iJhjHOiG_g3gAAH335jnDCo7crZJPiUMTLWUrQ8qJWk9wbCAbMjEaNanWuqNdkMfQ7h0Oq2AYgcDvVulS68AOG65g2vtjp1_wBaJjy2D0SqGoEibhxi5A9ehQxHUJmCUaTuNiHXGR02shFr8JF6wgP7K72OY4g58RKyOUnCDRoiYENuBcZzcT_dR2Zbmzyj9fqo7VBuMJv3ry4S1aM1JiUw2ctMabhAYbjFcp7DqhgGbl0bTbcQSpYiywDLyeJol7zU0tCgqT8KixXn1YbwphlgYBhw379KiMoBHuFzoQNjwsWRPK0lzgsNRE9j6KrCCwVEokSdrt9eTLHO5emM--J_F32JuF-eT__pQBBki0kU0nS0TZl-4TWY5L8L7cb8Awg-OEX4mxN_ONWAkxu3eSKuKPOWYUVwf-VVDhjHjfv74x8Eq84T4jqyXsMPmBZjmi_Xz5c2KUusC0fQYAJa5bP3FLjEd83IBA4n-B4fx1nVUtKZAZCkVXWgJNRrYRjZlY5kKHPfu9w-2T6Q_vLwNfr5FudEHlxfTAjecxGoFtgnRmCas1ZSf0UZ15FIAfO22mQlmiRdGZge_OXn4UuWOCLlRviBiZJTIbToKaYpgqlkqlg0j7nqx951xSCEMBjEL_GDi1IFZdm6RvQAPp=w830-h358-no?authuser=0)
```
Comment (noroadsleft) [inline keyboards/handwired/zergo/readme.md:3]: Please host this image somewhere publicly accessible, such as Imgur.

File: keyboards/handwired/zergo/readme.md [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+
+A compact 70% (15x6) keyboard inspired by the Katana60 project, but with F keys and adopted for the Russian layout.
+
+* Keyboard Maintainer: [Greenjack](https://github.com/greenjack-z)
+* Hardware Supported: own zErgo PCB for unique layout + chineese clone of Teensy 2.0++ controller.
```
Comment (noroadsleft) [inline keyboards/handwired/zergo/readme.md:8]:
```suggestion
* Hardware Supported: zErgo PCB, Teensy 2.0++
```

File: keyboards/handwired/zergo/readme.md [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+* Hardware Availability: protoptype self made hardware
```
Comment (noroadsleft) [inline keyboards/handwired/zergo/readme.md:9]:
```suggestion
* Hardware Availability: prototype self-made hardware
```

**Comment by noroadsleft** [comment]
Once `keyboards/handwired/zergo/makefile` is deleted, this should be good to merge IMO.

**Comment by noroadsleft** [comment]
Thanks!

For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

**Pull request merged**